src/libostree/ostree-repo.c

```diff
@@ -2528,6 +2528,7 @@ ostree_repo_remote_gpg_import (OstreeRepo         *self,
 #endif /* OSTREE_DISABLE_GPGME */
 }
 
+#ifndef OSTREE_DISABLE_GPGME
```

Comment:
Drive-by change: I was testing the `--with-gpme=no` path and there was an `unused-function` warning here.

Reply:
Could you sit that into a separate commit?

Reply:
Yup, done!